src/kirby/components/modal/services/modal.helper.ts

```diff
@@ -27,9 +41,10 @@ export class ModalHelper {
     return modal.onDidDismiss();
   }
 
-  private animate(
-    isAnimEnter: boolean,
+  private static animateIn(
     flavor: any,
```

Comment:
Consider not using "any" and use a type instead

Reply:
fixed